apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/store.ts

```diff
@@ -32,8 +32,10 @@ export const selectors = {
 	isSidebarClosing: ( state: State ) => state.closing,
 };
 
-registerStore( STORE_KEY, {
+export const store = createReduxStore( STORE_KEY, {
```

Comment:
I ended up adding back the STORE_KEY, because it's actually used in a different part of the codebase that doesn't make sense to change. This hack: https://github.com/Automattic/wp-calypso/blob/8cf39ff501a9ebb3b1af53832396804878b1352b/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L512-L520